lib/menubar/utils.lua

```diff
@@ -279,20 +267,21 @@ function utils.lookup_icon_uncached(icon_file)
         return false
     end
 
-    if icon_file:sub(1, 1) == '/' and is_format_supported(icon_file) then
+    local icon_file_ext = icon_file:gsub(".*%.","")
```

Comment:
Faster: `icon_file:match("^.+(%..+)$")` (1s instead of 1.4s with @psychon's benchmark).

Reply:
Nice work & thanks again